Update export documentation (#2023)
VirtusLab/scala-cli: website/docs/commands/export.md

# Exporting to JSON:

To export project information in a human-comprehensible format, use the `--json` flag.

> The default behaviour of export with `--json` provided is pritning to the standard output, this can be changed
> with `--output` by specifying a directory where to create the `export.json` file.

Reviewer: ```suggestion
By default, exporting with the `--json` option prints to standard output, this can be changed
with `--output` parameter by specifying a directory where to create the `export.json` file.
```
